pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -80,6 +80,61 @@ func NewNodeController(
 	}
 }
 
+// Generates num pod CIDRs that could be assigned to nodes.
+func generateCIDRs(num int) util.StringSet {
+	res := util.NewStringSet()
+	for i := 0; i < num; i++ {
+		// TODO: Make the CIDRs configurable.
+		res.Insert(fmt.Sprintf("10.244.%v.0/24", i))
```

Comment:
I know it can be considered a step back, because I'm deleting similar code from the bash script that might (if somebody had set some non-standard env parameters) have been taking care of this and just leaving out a TODO here. Note though that the nodecontroller still accepts nodes configured by someone else, especially those that already have the ip-range metadata set.
Are you suggesting to resolve this TODO in this PR or just mentioning it's important in general?